owl/srdl2-cap.owl

```diff
@@ -520,18 +522,18 @@
     
 
 
-    <!-- http://knowrob.org/kb/srdl2-cap.owl#VisualPerceptionAffordance -->
+    <!-- http://knowrob.org/kb/srdl2-cap.owl#VisualPerceptionCapability -->
 
-    <owl:Class rdf:about="http://knowrob.org/kb/srdl2-cap.owl#VisualPerceptionAffordance">
-        <rdfs:subClassOf rdf:resource="http://knowrob.org/kb/knowrob.owl#Affordance"/>
+    <owl:Class rdf:about="http://knowrob.org/kb/srdl2-cap.owl#VisualPerceptionCapability">
+        <rdfs:subClassOf rdf:resource="http://knowrob.org/kb/srdl2-cap.owl#PerceptionCapability"/>
     </owl:Class>
     
 
 
-    <!-- http://knowrob.org/kb/srdl2-cap.owl#VisualPerceptionCapability -->
+    <!-- http://knowrob.org/kb/srdl2-cap.owl#VisualPerceptionDisposition -->
```

Comment:
Is this disposition for a specific situation? E.g, for an agents standing at a specific pose the object may or may not be 
perceived  though the agent has the ability to perceive

Reply:
Hola. oh I the renaming was wrong actually.  I think it was meant as a general "CanSee" capability